app/networkapi/static/js/filebrowser/upload.js

```diff
@@ -132,19 +138,24 @@
                         .data('selectedFile', null);
                 }
 
+                // override: start #160
                 var results = form.find('.file-input-result.selected');
                 if(results.length==0) {
```

Comment:
note that in Javascript `==` is only equivalence, not identity, so you almost always want to use `===`  unless you really know why you're using `==`